tests/untldoc_test.py

```diff
@@ -712,3 +713,38 @@ def test_find_untl_errors_fix_errors_no_errors():
                                         'date': [{'content': '1944',
                                                   'qualifier': 'creation'}]},
                           'error_dict': {}}
+
+
+def test_untl_to_hash_dict():
```

Comment:
So I think we need to parametrize this test to also show that `meaningfulMeta` is doing its job.